rebound/particle.py

```diff
@@ -19,6 +19,7 @@ class Particle(Structure):
     The main REBOUND particle data structure. 
     This is an abstraction of the reb_particle structure in C.
     The Particle fields are set at the end of simulation.py to avoid circular references.
+    The Keplerian Elements are in Jacobi coordinates (with mu = G*Minc, where Minc is the total mass from index 0 to the particle's index, inclusive).
```

Comment:
Ok. Would you like me to put them all in one line on the table as well? I was originally afraid that it would mess up the look and formatting of the table which is why I spread them across two different rows. I can put it all in one row at the bottom, especially if the table automatically does wrap around text in the cells.

Reply:
No strong opinion there. One row would make more sense conceptually...

Reply:
Okay, that's what I originally thought as well. I'll put it all in one row then. I think it would look better an make more sense that way too.